mandatory/3-function-output.js

```diff
@@ -1,18 +1,30 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+// This 'getNumber' function will return a random number between 0 (inclusive) and 10 (exclusive), which cannot be chosen or reset by the user.
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+// This 's' function with the .concat() method is used to merge two or more arrays, by returning a new array and not changing the existing arrays.
 function s(w1, w2) {
   return w1.concat(w2);
```

Comment:
Are you sure the ``concat`` method is used to merge arrays?

Reply:
sorry, bad use of terminology; it has now been changed :-)